lms/djangoapps/ccx/views.py

```diff
@@ -574,6 +585,13 @@ def ccx_grades_csv(request, course, ccx=None):
             courseenrollment__course_id=ccx_key,
             courseenrollment__is_active=1
         ).order_by('username').select_related("profile")
+
+        if configuration_helpers.get_value('HIDE_MASTER_COURSE_STAFF_FROM_GRADEBOOK', False):
```

Comment:
Probably here we could also use `LicensedEnrollments`, In fact we could replace lines 585-886 with
```
licenseenrollment__courseenrollment__course_id=ccx_key,
licenseenrollment__courseenrollment__is_active=1
```